DEVELOPMENT.md

```diff
@@ -0,0 +1,26 @@
+# How to prepare a development environment
+
+1. [Install poetry](https://python-poetry.org/docs/#installation):
+```shell
+# MacOS / Linux
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+# Windows
+(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python -
+```
+2. Install dependencies:
+```shell
+poetry install
+```
+3. (Optional) Activate the poetry virtual environment:
+```shell
+poetry shell
+```
+4. Run tests:
+```shell
+pytest
```

Comment:
It can be done by running `tox` actually. It will run all tests and mypy, not linters though.
Linters can be run using `pre-commit run --all-files`

I actually wanted to get rid of the Makefile.

Reply:
Ok, sounds fine as long as it's documented in `CONTRIBUTING.md` or in our case in this file.
So maybe let's add point 5 about installing and running `pre-commit`.

Reply:
Done, I added the `pre-commit install` in the setup step